TwitterKit.json

```diff
@@ -1,4 +1,3 @@
 {
-	"3.2.1": "https://ton.twimg.com/syndication/twitterkit/ios/3.2.1/Dynamic/TwitterKit.framework.zip",
-	"3.3.0": "https://ton.twimg.com/syndication/twitterkit/ios/3.3.0/Dynamic/TwitterKit.framework.zip"
+	"3.3.0": "https://github.com/abema/twitter-kit-ios/raw/develop/TwitterKit/Products/3.3.0/framework.zip"
```

Comment:
It's not a good idea to commit the binary.
We should update them in GitHub releases page manually. (I wouldn't recommend to automate the upload process. API is not reliable.)

Reply:
Remove all zip files and add to .gitignore file.